src/agent/agent.c

```diff
@@ -1932,6 +2004,7 @@ static const sd_bus_vtable agent_vtable[] = {
                         agent_property_get_disconnect_timestamp,
                         0,
                         SD_BUS_VTABLE_PROPERTY_EXPLICIT),
+        SD_BUS_PROPERTY("LastSeenTimestamp", "t", agent_property_get_last_seen, 0, SD_BUS_VTABLE_PROPERTY_EXPLICIT),
```

Comment:
You can remove the function `agent_property_get_last_seen` and use `offsetof` since the property is just a simple field on the `Agent` struct. 
(This could then also be refactored for `DisconnectTimestamp` if you are up for it)

```suggestion
        SD_BUS_PROPERTY("LastSeenTimestamp", "t", NULL, offsetof(Agent, connection_last_seen), SD_BUS_VTABLE_PROPERTY_EXPLICIT),
```

Reply:
done